<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Title">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RSA Cipher Application</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Blablabla">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>770</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>531</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_Type_Text">
       <property name="text">
        <string>Type your text here : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_Type_Text"/>
     </item>
     <item>
      <widget class="QLabel" name="label_New_Text">
       <property name="text">
        <string>New Text :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_New_Text"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_Generate_Keys">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>700</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate a key pair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Encrypt">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>700</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt the message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Decrypt">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>700</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt the message </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Private_Key">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>109</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>Private key pair :       </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Public_Key">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Public key pair :</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>471</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Public_N_Label">
       <property name="text">
        <string> n =   </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Public_N"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Public_E_Label">
       <property name="text">
        <string> e =  </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Public_E">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>471</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="Private_N_Label">
       <property name="text">
        <string> n =   </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Private_N"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Private_D_Label">
       <property name="text">
        <string> d = </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Private_D">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
